app/main.py

```diff
@@ -2,4 +2,77 @@ class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(
+            self,
+            name: str,
+            weight: int,
+            coords: list = None
+    ) -> None:
+        self.name = name
+        self.weight = weight
+        self.coords = coords if coords is not None else [0, 0]
```

Comment:
You could remove `is not None` here
```suggestion
        self.coords = coords if coords else [0, 0]
```